superset/assets/src/setup/setupApp.js

```diff
@@ -20,6 +20,16 @@
 import $ from 'jquery';
 import { SupersetClient } from '@superset-ui/connection';
 import getClientErrorObject from '../utils/getClientErrorObject';
+import moment from 'moment';
+import localStorage from 'local-storage';
+
+const DEFAULT_D3_LANGUAGE = 'en';
+
+function applyLocales() {
+  const currentLocale = localStorage.get('locale');
```

Comment:
Where is the code that set the `locale`?

Reply:
Nice catch, I haven't included it - I'll add it in next commit there: https://github.com/apache/incubator-superset/blob/1dd4d7a587a55525ff14e1e051d268adccbc15a1/superset/assets/src/preamble.js#L31